packages/rocketchat-livechat/server/lib/QueueMethods.js

```diff
@@ -131,5 +131,63 @@ RocketChat.QueueMethods = {
 		RocketChat.models.Rooms.insert(room);
 
 		return room;
+	},
+	'Bot_First'(guest, message, roomInfo) {
```

Comment:
what you think instead of duplicating the code create a new function with the both behaviors (changeable by parameters) and just calling it in both queue methods?